imap_processing/swe/l1b/swe_l1b_science.py

```diff
@@ -108,12 +108,9 @@ def deadtime_correction(counts: np.ndarray, acq_duration: int) -> npt.NDArray:
     corrected_count : numpy.ndarray
         Corrected counts.
     """
-    # deadtime will be constant once it's defined.
-    # This deadtime value is from previous mission. SWE
-    # will give new one once they have it ready.
-    # TODO: update deadtime when we get new number
-    deadtime = 1.5e-6
-    correct = 1.0 - (deadtime * counts / acq_duration)
+    # deadtime is 360 ns
+    deadtime = 360e-9 / 1e9  # convert to seconds
```

Comment:
I think this should just be, don't divide it by 1e9 again. This should get a test so we know it is being applied correctly.
```suggestion
    deadtime = 360e-9
```

Reply:
Thank you for catching this. I will check with Ruth about this again. She sent me this `SWE deadtime: 360 ns` and now when I am calculating corrected counts, I need to convert deadtime and acquisition times to seconds to be consistent. That's why I think I did that. But good point. I will check with Ruth.

Reply:
Yes, 360e-9 is 360 ns. You could write it as 360 / 1e9 too. I don't think you need to check this I'm confident it should be this.